tutorials/docker/Dockerfile

```diff
@@ -1,27 +1,59 @@
-FROM rticom/connext-sdk:7.3.0-EAR as build-stage
+ARG RTI_LICENSE_AGREEMENT_ACCEPTED=false
```

Comment:
why is this `ARG` defined here, before FROM and again in line 4?

Reply:
I got some information for how to design my dockerfile based on this: [https://bitbucket.rti.com/projects/PM/repos/tsn-arch-sim.git/browse/docker/Dockerfile]

I also spoke to Marco in the help-deb channel because previously I only had the ARG defined once inside the build-stage with a default value of false, and with my command line argument, the value was not changing. We thought that it needed to be defined with a default outside of the build-stage and then defined without a value inside the build stage to either inherit the command-line value or default value because previously it was only defined once in the build-stage and always using the default value. 

But later down the line I realized we were using the wrong value. It should have been "accepted" instead of "yes" or "true".

I just reran it by only declaring it once with a default in build-stage and it built fine. So I'll remove that